README.md

```diff
@@ -49,13 +49,13 @@ jobs:
 
 ## Versioning
 
-Using the versioned semantic [tags](#Tags) is recommended for security and reliability.
+Using the versioned semantic [tags](#tags) is recommended for security and reliability.
 
 See [GitHub: Using tags for release management](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management)
 and [GitHub: Keeping your actions up to date with Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot)
 .
 
-For convenience, it is possible to use the [branches](#Branches) following the major releases.
+For convenience, it is possible to use the [branches](#branches) following the major releases.
```

Comment:
This change is not correct. The anchor name has a capital.

Reply:
@julien-carsique-sonarsource did you try? 

I did, and it actually works as expected; that issue was reported by the markdown linter.